onnxruntime/core/providers/nuphar/compiler/x86/scheduler/tvm_rule_schedule.cc

```diff
@@ -81,8 +84,13 @@ static bool ReduceVScheduleNupharX86(
 
         ctx_sched.schedule[tensor->op].reorder(reorder_axis);
         ctx_sched.schedule[tensor->op].unroll(x0);
+        try_parallel = false;
```

Comment:
Parallel should have higher priority than unrolling when possible. 
Here unrolling is for the head dim, which is reordered into the most inner dim. 
This make sense for single thread case, but for multi-thread case, parallel head dim should have more benefits than unrolling when possible.

Reply:
Will look at that in next PR